src/inc/manuscriptUtils.ts

```diff
@@ -0,0 +1,25 @@
+import { iManuscript, iManuscriptPopup } from '../types/types';
+
+export const getImageUrl = (manuscripts: iManuscript[], sublineData: iManuscriptPopup): string => {
+  return (
+    manuscripts?.find(
+      (manuscript) =>
+        manuscript?.slug === 'leiden' &&
```

Comment:
"leiden" shouldn't be hard-coded but should come from the button clicked.
Later we will have other manuscripts, like "kricha"